data_sources.py

```diff
@@ -6,9 +6,13 @@
                               "C2BYNO01_T_N.csv.zip",
                               "C2CBUD01_T_N.csv.zip",
                  ],
+                 "Stredocesky": [
+                              "P3HULI01_T_N.csv.zip",
+                              "P3NETV01_T_N.csv.zip",
+                 ],
                  "Liberecky": [
                               "U2HRAD01_T_N.csv.zip",
-			      "U2STRR01_T_N.csv.zip",
+			                  "U2STRR01_T_N.csv.zip",
```

Comment:
Díky za commit stredoceskeho. 
Vyladíme jeden detail a můžeme to spojit s main větví.

Ten malý detail je, že kromě přidání řádků 9 až 12 došlo i ke změně na řádku 15 s textem
"U2STRR01_T_N.csv.zip", kde přibyly mezery.

Ten řádek by to chtělo vrátit do původní podoby a pak znovu 
add / commit / pull